index.js

```diff
@@ -0,0 +1,17 @@
+const menuBars = document.querySelector('.menu-bars');
+const mobileMenuContainer = document.querySelector('#mobile-menu-container');
+const mobileCloseMenu = document.querySelector('#mobile-close-menu');
+
+const mobileNav = document.querySelector('#mobile-menu-container>ul>li>a');
+
+menuBars.addEventListener('click', () => {
+  mobileMenuContainer.style.display = 'flex';
+});
+
+mobileCloseMenu.addEventListener('click', () => {
+  mobileMenuContainer.style.display = 'none';
+});
+
+mobileNav.addEventListener('click', () => {
+  mobileMenuContainer.style.display = 'none';
+});
```

Comment:
# [OPTIONAL]
This kind of operations are hard to understand for someone that didn't write the code! Let me elaborate!
What does changing the style to flex does? I have no idea! But I have to guess this is the way you control the open and close menu events! Lucky for you JS scripts counts with really powerful methods that make this kind of process easy to follow and maintain! Let me present you the [toggle](https://www.w3schools.com/howto/howto_js_toggle_class.asp)! method!

I would kindly suggest as a personal challenge to try to refactor your menu so its default state is closed and you can toggle the open state! your final code would look something like this!: 
```suggestion

mobileCloseMenu.addEventListener('click', () => {
  myMenuButton.classList.toggle = ("open");
});

```

As you can see it makes a lot more semantic sense to have a sentence that tells you it `toggles open` a menu. Of course this change will require some refactor So only try to do it if you are finished with all your other activities!